### PR TITLE
Update jquery mobile

### PR DESCRIPTION
**Pull request by matthewdunsdon**
Relates to #310

### CODE REVIEW

**Comment by ColinEberhardt** [comment]
Nice work - just tried this out and it works great :-)

Probably just needs the README updating and then its ready to merge.

File: jquerymobile/www/index.html [hunk 1]
```diff
@@ -1,23 +1,149 @@
...
+  <script type="text/javascript" src="lib/jquery.mobile-1.4.5.min.js"></script>
+  <script type="text/javascript" src="lib/knockout-3.3.0.js"></script>
+  <!-- Automatically included by PhoneGap Build -->
+  <script type="text/javascript" src="cordova.js"></script>
+  <script type="text/javascript" src="lib/require.js" data-main="app"></script>
+
+  <!-- jquery mobile styling -->
+  <link rel="stylesheet" href="lib/jquery.mobile-1.4.5.min.css" />
+  <!-- application specific styling -->
+  <link rel="stylesheet" type="text/css" href="style/style.css" />
```
Comment (arthurvr) [inline jquerymobile/www/index.html]: remove the `type="text/css"`

File: jquerymobile/www/index.html [hunk 2]
```diff
@@ -1,23 +1,149 @@
...
+  <meta name="viewport" content="width=device-width, user-scalable=no" />
+
+  <!-- hide the navigation bar when bookmarked on an iPhone -->
+  <meta name="apple-mobile-web-app-capable" content="yes" />
+
+  <script type="text/javascript" src="lib/jquery-1.11.1.min.js"></script>
+  <script type="text/javascript" src="lib/jquery.mobile-1.4.5.min.js"></script>
+  <script type="text/javascript" src="lib/knockout-3.3.0.js"></script>
+  <!-- Automatically included by PhoneGap Build -->
+  <script type="text/javascript" src="cordova.js"></script>
```
Comment (arthurvr) [inline jquerymobile/www/index.html]: remove all those `type="text/javascript"`

**Comment by arthurvr** [comment]
Sorry for the nitpicks ;) nice work!

**Comment by ColinEberhardt** [comment]
> Sorry for the nitpicks ;) nice work!

Thanks @arthurvr - any additional reviews are welcome :-)

**Comment by matthewdunsdon** [comment]
@ColinEberhardt Are there any outstanding issues on this pull request?

The README file has been cleaned up to reflect the changes made to the project (i.e. the move to using the default theme).

**Comment by matthewdunsdon** [comment]
One outstanding issue is the JQueryMobile assets, which are using an old JQueryMobile logo.  Should this be fixed as par of this PR?

It is worth noting, the website PR includes an update to the JQueryMobile logo (https://github.com/matthewdunsdon/PropertyCross-Site/commit/f5254b66b90a8a0e6609121650468be003989ed1)

**Comment by ColinEberhardt** [comment]
Great stuff - while you are working in this area, it would be great if you could update the logo too. I'll merge immediately after you do that.

Thanks for contributing :-)

**Comment by matthewdunsdon** [comment]
Updated JQueryMobile icons

- Also updated titanium icon and asset generation scripts

**Comment by ColinEberhardt** [comment]
Awesome - thanks. I'll take a look at your 'site' PR next

**Pull request merged**